app/domain/operations/hbx_enrollments/drop_enrollment_members.rb

```diff
@@ -19,6 +19,7 @@ def call(params)
       def validate(params)
         return Failure('Missing HbxEnrollment Key.') unless params.key?(:hbx_enrollment)
         return Failure('Not a valid HbxEnrollment object.') unless params[:hbx_enrollment].is_a?(HbxEnrollment)
+        return Failure('Not an ivl enrollment.') unless params[:hbx_enrollment].is_ivl_by_kind? # Added per the request, also need further development if to be used for shop
         return Failure('No members selected to drop.') if params[:options].select{|string| string.include?("terminate_member")}.empty?
         return Failure('No termination date given.') if params[:options].select{|string| string.include?("termination_date")}.empty?
 
```

Comment:
Missing couple of checks,

return Failure('Can't Drop subscriber of enrollment .')
return Failure('Enrollment need be in active state to drop dependent')
return Failure("Check whether feature enabled or not ")

This operation should work independent of UI, should drop dependent when necessary params to this operation when called outside

Reply:
It was requested to be able to drop the subscriber from enrollment. 
I don't think we check if the feature is enabled inside the operation so that the operation can be used from the backend if needed. I can add that if you still want me to. 
The rest of the things are being blocked through UI, but will add them here.

Reply:
you can skip this return Failure('Can't Drop subscriber of enrollment .')

please add rest other failures. This operation should work independent of UI

Reply:
Working on adding the changes.